cosmos/providers/dbt/core/operators_kubernetes.py

```diff
@@ -0,0 +1,389 @@
+
+        global_boolean_flags = [
+            "no_version_check",
+            "cache_selected_only",
+            "fail_fast",
+            "quiet",
+            "warn_error",
+        ]
+        for global_boolean_flag in global_boolean_flags:
+            dbt_name = f"--{global_boolean_flag.replace('_', '-')}"
+            global_boolean_flag_value = self.__getattribute__(global_boolean_flag)
+            if global_boolean_flag_value is True:
+                flags.append(dbt_name)
+        return flags
+
+    def build_env_args(self, env: dict):
+        env_vars_dict = {}
+        for env_var in self.env_vars:
+            env_vars_dict[env_var.name] = env_var.value
+
+        self.env_vars = convert_env_vars({**env, **env_vars_dict})
+
+    def build_cmd(self, env: dict, cmd_flags: list = None):
+        profile_vars = {}
+        if conn_exists(conn_id=self.conn_id):
+            _, profile_vars = map_profile(
+                conn_id=self.conn_id,
+                db_override=self.db_name,
+                schema_override=self.schema,
+            )
+
+        # parse dbt args
+        dbt_args = []
+
+        ## start with the dbt executable
+        self.cmds = [self.dbt_executable_path]
+
+        ## add base cmd
+        if isinstance(self.base_cmd, str):
+            dbt_args.append(self.base_cmd)
+        else:
+            [dbt_args.append(item) for item in self.base_cmd]
+
+        # add global flags
+        for item in self.add_global_flags():
+            dbt_args.append(item)
+
+        ## add command specific flags
+        if cmd_flags:
+            for item in cmd_flags:
+                dbt_args.append(item)
+
+        ## set env vars
+        self.build_env_args({**env, **profile_vars})
+
+        self.arguments = dbt_args
+
+
+class DbtLSOperator(DbtKubernetesBaseOperator):
+    """
+    Executes a dbt core ls command.
+
+    """
+
+    ui_color = "#DBCDF6"
+
+    def __init__(self, **kwargs) -> None:
+        super().__init__(**kwargs)
+        self.base_cmd = "ls"
+
+    def execute(self, context: Context):
+        self.build_cmd(env=self.get_env(context))
+        return super().execute(context)
+
+
+class DbtSeedOperator(DbtKubernetesBaseOperator):
+    """
+    Executes a dbt core seed command.
+
+    :param full_refresh: dbt optional arg - dbt will treat incremental models as table models
+
+    """
+
+    ui_color = "#F58D7E"
+
+    def __init__(self, full_refresh: bool = False, **kwargs) -> None:
+        self.full_refresh = full_refresh
+        super().__init__(**kwargs)
+        self.base_cmd = "seed"
+
+    def add_cmd_flags(self):
+        flags = []
+        if self.full_refresh is True:
+            flags.append("--full-refresh")
+
+        return flags
+
+    def execute(self, context: Context):
+        cmd_flags = self.add_cmd_flags()
+        self.build_cmd(env=self.get_env(context), cmd_flags=cmd_flags)
+        return super().execute(context)
```

Comment:
I'm not sure about the naming of these classes (they inherit from the `DbtKubernetesBaseOperator` but they don't have the word kubernetes in them).

Perhaps we should have different "modes" in the DbtKubernetesBaseOperator. WDYT?

Reply:
I think it's a good idea. I made the change. I also changed the base operator by adding Local to specify that, unlike the other two, the dbt will be run locally. 

Maybe I shouldn't have or should have put Bash instead. WDYT?